pkg/profile/artifact_single_repo_test.go

```diff
@@ -0,0 +1,143 @@
+package profile_test
```

Comment:
how come we moved this test to a separate file?

Reply:
To make artifact_test shorter. :D

Reply:
I think thats a smell :wink: I think I would naturally look for this test in `artifact_test.go` since its testing `artifact.go` logic. I think we can do a lot of other work to make the `artifact_tests` smaller https://github.com/weaveworks/pctl/pull/144#discussion_r653552254